docassemble/ALKilnInThePlayground/data/static/alkiln_playground.css

```diff
@@ -85,17 +194,35 @@ iframe#alkiln_notifications {
   width: 100%;
 }
 
-.accordion.output {
-  background: var(--bs-accordion-active-bg);
+.alk_no-gutters > * {
+  padding: 0;
+}
+
+.quick_buttons.top {
+  margin-bottom: 3rem;
+}
```

Comment:
Let's see if there's a var bootstrap is using for the top and use that instead if we can